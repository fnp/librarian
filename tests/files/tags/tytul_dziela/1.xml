<akap>
  Przeczytałem wczoraj
  <tytul_dziela>
    Kilka uwag o
    <tytul_dziela typ="1">Hamlecie</tytul_dziela>
    Szekspira
  </tytul_dziela>
  pióra...
</akap>
