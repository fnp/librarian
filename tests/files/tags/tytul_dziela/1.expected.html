<div id="book-text">
<a href="#f1" class="wl-num">1</a><p class="wl paragraph" id="f1">Przeczytałem wczoraj
  <em class="book-title">
    Kilka uwag o <em class="book-title">„Hamlecie”</em>
    Szekspira
  </em>
  pióra...</p>
</div>
